public-api/db-cdr/generate-cdr/make-bq-data.sh

```diff
@@ -614,19 +614,19 @@ group by d2.domain_id,c.vocabulary_id"
 ######################################
 echo "Updating rolled up counts of snomed conditions in concept"
 bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
-"Update \`$OUTPUT_PROJECT.$OUTPUT_DATASET.concept\` c
-set c.count_value=sub_cr.est_count
-from (select concept_id, est_count from \`$OUTPUT_PROJECT.$OUTPUT_DATASET.criteria\` cr
-where cr.concept_id = concept_id and cr.type='SNOMED' and cr.subtype='CM' and cr.synonyms like '%rank1%') as sub_cr
-where sub_cr.concept_id = c.concept_id and c.domain_id = 'Condition' and c.vocabulary_id = 'SNOMED' "
+"UPDATE \`$OUTPUT_PROJECT.$OUTPUT_DATASET.concept\` c
```

Comment:
Not sure why the above query is not updating but the new update join query works fine to copy over rolled up counts from criteria to concept.